### PR TITLE
Fix #4

### LINKED ISSUE #4: Trimmer returns 0 even if the build failed
I'm using Trimmer as part of my build pipeline. Sometimes when the build fails, Unity returns 0 even when it shouldn't. This is on Ubuntu 2019.3.0f5. I'm not sure when exactly this happens, though. What info can I give you to help narrow this down?

### PR DESCRIPTION
**Pull request by JesseTG**

### CODE REVIEW

**Comment by JesseTG** [comment]
Wait, no, I think I found a simpler way.

File: Editor/BuildManager.cs [hunk 1]
```diff
@@ -286,18 +302,10 @@ public static string Build()
         string result;
```
Comment (sttz) [inline Editor/BuildManager.cs:302]: You can remove this variable now since it's no longer used

File: Editor/BuildManager.cs [hunk 2]
```diff
@@ -215,25 +215,49 @@ public static void UnityCloudBuild(BuildManifestObject manifest)
...
+                // If an error was returned..
+                Debug.LogError(result);
+            }
+
+            if (IsCommandLineBuild) {
+                //
+                throw new BuildFailedException(result);
+            }
+        }
+        finally {
```
Comment (sttz) [inline Editor/BuildManager.cs:251]: Put the braces and the finally on the same line: "} finally {"

**Review by sttz — CHANGES_REQUESTED** [review]
Great, thanks for the PR!

Looks good overall. Try to keep whitespace changes to a minimum and put them in a separate commit, makes reviewing much easier.

**Comment by JesseTG** [comment]
Ah, it's not ready yet. I gotta test this locally first.

**Pull request closed without merging**